# 四足机器狗的舵机目标角度，各数组长度为3
float32[] servo_FL  # 左前腿
float32[] servo_FR  # 右前腿
float32[] servo_BL  # 左后腿
float32[] servo_BR  # 右后腿